apps/site/pages/en/learn/getting-started/debugging.md

```diff
@@ -129,6 +129,7 @@ The following table lists the impact of various runtime flags on debugging:
 | --inspect-brk=[host:port]          | Enable inspector agent; Bind to address or hostname host (default: 127.0.0.1); Listen on port port (default: 9229); Break before user code starts     |
 | --inspect-wait                     | Enable inspector agent; Listen on default address and port (127.0.0.1:9229); Wait for debugger to be attached.                                        |
 | --inspect-wait=[host:port]         | Enable inspector agent; Bind to address or hostname host (default: 127.0.0.1); Listen on port port (default: 9229); Wait for debugger to be attached. |
+| --disable-sigusr1                  | Disable the ability of starting a debugging session by sending a SIGUSR1 signal to the process (since Node v23.7.0, v22.14.0).                        |
```

Comment:
```suggestion
| --disable-sigusr1                  | Disable the ability of starting a debugging session by sending a SIGUSR1 signal to the process.                        |
```

For the most part, we assume the user is using the latest version of Node.js

Reply:
@avivkeller apologies, I missed your suggestion. It's committed now.